.rubocop_todo.yml

```diff
@@ -8,21 +9,50 @@
 # Offense count: 25
 # Configuration parameters: AllowSafeAssignment.
 Lint/AssignmentInCondition:
-  Enabled: false
+  Exclude:
+    - 'lib/fog/exoscale/models/compute/flavors.rb'
+    - 'lib/fog/exoscale/models/compute/images.rb'
+    - 'lib/fog/exoscale/models/compute/job.rb'
+    - 'lib/fog/exoscale/models/compute/jobs.rb'
+    - 'lib/fog/exoscale/models/compute/security_groups.rb'
+    - 'lib/fog/exoscale/models/compute/zones.rb'
+    - 'lib/fog/exoscale/requests/compute/authorize_security_group_egress.rb'
+    - 'lib/fog/exoscale/requests/compute/authorize_security_group_ingress.rb'
+    - 'lib/fog/exoscale/requests/compute/deploy_virtual_machine.rb'
+    - 'lib/fog/exoscale/requests/compute/list_security_groups.rb'
+    - 'lib/fog/exoscale/requests/compute/list_service_offerings.rb'
+    - 'lib/fog/exoscale/requests/compute/query_async_job_result.rb'
+    - 'lib/fog/exoscale/requests/compute/revoke_security_group_egress.rb'
+    - 'lib/fog/exoscale/requests/compute/revoke_security_group_ingress.rb'
```

Comment:
I worry that this wont help at all. The code will still be bad. Can we leave as is and let rubocop break until we fix it?